tests/bettertransformer/Dockerfile_bettertransformer_gpu

```diff
@@ -0,0 +1,25 @@
+FROM nvidia/cuda:11.7.1-cudnn8-devel-ubuntu22.04
+CMD nvidia-smi
+
+# Ignore interactive questions during `docker build`
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install and update tools to minimize security vulnerabilities
+RUN apt-get update
+RUN apt-get install -y software-properties-common wget apt-utils patchelf git libprotobuf-dev protobuf-compiler cmake \
+    bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 mercurial subversion libopenmpi-dev && \
+    apt-get clean
+RUN unattended-upgrade
+RUN apt-get autoremove -y
+
+RUN python -m pip install -U pip
+
+RUN pip install torch torchvision torchaudio
+RUN pip install transformers accelerate
```

Comment:
```suggestion
RUN pip install transformers accelerate
```
Maybe we should install `accelerate` from source? No strong opinion but when doing this in `transformers` it helps avoiding to have some issues before `accelerate` releases. Wdyt?

Reply:
SGTM !